docs/development/release_management.rst

```diff
@@ -67,8 +67,10 @@ Pre-Release
 9. Push the branch and tags.
 10. Build Debian packages and place them on ``apt-test.freedom.press``. This is currently done
     by making a PR into `a git-lfs repo here <https://github.com/freedomofpress/securedrop-dev-packages-lfs>`_.
-    Changes merged to ``master`` in this repo will be published within 15 minutes. Build
-    logs should be saved and published according to the
+    Only commit packages that are new: do not clobber existing packages. That is, if there is already
+    a deb called e.g. ``ossec-agent-3.0.0-amd64.deb`` in ``master``, do not commit a new version of this deb.
+    Updated debian packages must have an different version. Changes merged to ``master`` in this repo
+    will be published within 15 minutes. Build logs should be saved and published according to the
```

Comment:
This list item has a lot going on now - would it be worth splitting publishing the build log into a separate item?